Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: *
+
+all: build-base build-env 
+
+build-base:
+	git clone --depth 1 https://github.com/sunil-sahoo/opencart.git -b docker-deploy /project
+	docker build -t build-env -f /project/Dockerfile.build /project
+
+build-env:
+	docker run -it --rm --name ubuntu-build-env -v /var/run/docker.sock:/var/run/docker.sock -v /project:/project build-env
+
+reset:
+	docker exec -it opencart mv /project/upload/config-dist.php /project/upload/config.php 
```

Comment:
This should be executed as a part of image build or BEFORE the docker-compose start

Reply:
docker exec -it opencart chown -R www-data:www-data /var/www/html
Is this the only command for "make reset" ?

Reply:
Make reset is when you already started compose and need to call API on it

Reply:
Right now i don't have anything for "make reset", so should i left it empty?